### ISSUE TITLE
Ability to change app language independent of system language

### ISSUE THREAD

**Issue by GithubUser232** [martykan/forecastie#666]

**Comment by robinpaulson** [comment]
This came up previously, in #134. Could you take a look and see if that fulfils your needs?

**Comment by GithubUser232** [comment]
> This came up previously, in #134. Could you take a look and see if that fulfils your needs?

Not exactly. #134 wants this option for languages which are not present on Android but my intention is just an option independent of Android for all language that app support.

**Comment by robinpaulson** [comment]
Ah, interesting. I'm curious, in which situations would a person use a particular app in a different language to the rest of the system?

**Comment by GithubUser232** [comment]
> Ah, interesting. I'm curious, in which situations would a person use a particular app in a different language to the rest of the system?

I think this [article](https://localazy.com/blog/why-allow-users-to-switch-languages) can help.

**Comment by robinpaulson** [comment]
> > Ah, interesting. I'm curious, in which situations would a person use a particular app in a different language to the rest of the system?
> 
> I think this [article](https://localazy.com/blog/why-allow-users-to-switch-languages) can help.

Fascinating

**Comment by robinpaulson** [comment]
As an aside, localazy looks like an interesting platform, what experience do you have using it?

**Comment by GithubUser232** [comment]
> As an aside, localazy looks like an interesting platform, what experience do you have using it?

Yes it sounds good but I have no experience, I wish i had. I will try to use it as a tester.

**Comment by robinpaulson** [comment]
Any information on how to implement language selection would be welcome, it strikes me as slightly outside the usual Android ecosystem. I wonder if @verdulo might have found anything relevant during their research?

**Comment by Korb** [comment]
> I'm curious, in which situations would a person use a particular app in a different language to the rest of the system?

For me, my native language is Russian. And I don't speak any other languages, I just read a little English and French. I prefer the language of the operating system, be it a computer or a smartphone, to be in Russian. However, there are three points:

1. The inscriptions in Russian are very long. If there are too many, they take up too much space on the screen. This is critical for mobile devices. Therefore, I prefer that applications whose interface I am familiar with are in English. Especially on a smartphone. Yes, inscriptions in Russian are often abbreviated. And because of this, they become obscure.

2. Most often, the developer adheres to the unity of terminology in their applications. That is, the same things in different places of the application (and the help system for it, and in different versions of the application) are called the same. However, translators can break this rule, which makes it difficult to understand how applications work and master them. In addition, in one version of the application, one place could be translated in one way, and in another version - differently. This happens when the applications are large, with tens or hundreds of people working on them. Like expensive professional packages and operating systems. This leads to the complexity of changing the version of the application by the user.

3. There are very few Russian-speaking Internet users compared to the number of English-speaking users. So, using English on the Internet is more likely to find a solution to the problem. Of course, when searching, you need to use the English-language inscriptions of the application and English-language error messages.

4. Agree that posting here a screenshot with the Russian-language interface of your application or a screenshot of an error message in Russian to you, as the developer of this application, would not be as useful as these screenshots in English.
